File: FlexServ Backend/DotNet/AdminService/Dtos/AddressResponse.cs
namespace AdminService.Dtos
{
    public class AddressResponse
    {
        public long Id { get; set; }
        public string? HouseNo { get; set; }
        public string? Street { get; set; }
        public string? Area { get; set; }
        public string? City { get; set; }
        public string? State { get; set; }
        public string? Pincode { get; set; }

        public AddressResponse() { }

        public AddressResponse(long id, string? houseNo, string? street, string? area, string? city, string? state, string? pincode)
        {
            Id = id;
            HouseNo = houseNo;
            Street = street;
            Area = area;
            City = city;
            State = state;
            Pincode = pincode;
        }
    }
}

File: FlexServ Backend/SpringBoot/BusinessService/src/main/java/com/flexserv/dto/response/AddressResponse.java
package com.flexserv.dto.response;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class AddressResponse {

    private Long id;
    private String houseNo;
    private String street;
    private String area;
    private String city;
    private String state;
    private String pincode;
}